modules/core/shared/src/main/scala/scalacache/CacheAlg.scala

```diff
@@ -83,10 +73,8 @@ trait CacheAlg[V] {
     * It will take care of gracefully shutting down the underlying cache client.
     *
     * Note that you should not try to use this Cache instance after you have called this method.
-    *
-    * @param mode The operation mode, which decides the type of container in which to wrap the result
-    * @tparam F The type of container in which the result will be wrapped. This is decided by the mode.
     */
-  def close[F[_]]()(implicit mode: Mode[F]): F[Any]
+  //TODO: Replace with Resource-based API?
```

Comment:
Possibly, but let's save that for another PR

Reply:
sure